### PR TITLE
add ludo-game

### PR DESCRIPTION
**Pull request by Gurnav224**
## PR Description 📜
I add a ludo game

Fixes 

<hr>
 
## Mark the task you have completed ✅

- [ ✅] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ ✅] I have performed a self-review of my own code or work.
- [ ✅] I have commented my code, particularly in hard-to-understand areas.
- [✅ ] My changes generates no new warnings.
- [✅ ] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [ ✅] I have added screenshot for website preview in assets/images 
- [ ✅] I have added entries for my game in main README.md
- [ ] I have added README.md in my folder 
- [ ] I have added working video of the game in README.md (optional)
- [ ] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

File: README.md
Comment (kunjgit) [inline README.md:1]: don't add number over here and make sure you mention * as number as we are adding games it will conflicts !! 
make the changes !!

**Review by kunjgit — CHANGES_REQUESTED** [review]

File: README.md
Comment (Gurnav224) [reply README.md:1]: i can't understand where to remove the number can you please tell where to remove the number

**Comment by kunjgit** [comment]
please mention the issue on which you were working and make a PR again !!

**Comment by kunjgit** [comment]
Thank you  @Gurnav224 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request closed without merging**